server_test.go

```diff
@@ -402,10 +402,10 @@ func TestGroupParams(t *testing.T) {
 	t.Log(document.Paths.Find("/").Get.Parameters[0].Value.Name)
 	require.Len(t, document.Paths.Find("/").Get.Parameters, 1)
 	require.Equal(t, document.Paths.Find("/").Get.Parameters[0].Value.Name, "Accept")
```

Comment:
Not your change, but we may want to check if `require.NotNil(t, document.Paths.Find("/").Get.Parameters[0].Value)`

Also could probably change all the `require.Equal` below to `assert`.

Reply:
This is just a nitpick feel free to resolve